pyartcd/pyartcd/pipelines/brew_scan_osh.py

```diff
@@ -0,0 +1,75 @@
+"""
+[STORES STATE IN REDIS]
+This job scans the candidate tags for a particular version, and triggers scans for builds that are tagged into it.
+"""
+import click
+from typing import Optional
+from pyartcd import exectools, redis
+from pyartcd.runtime import Runtime
+from pyartcd.cli import cli, pass_runtime, click_coroutine
+
+
+class OshScan:
+    def __init__(self, runtime: Runtime, version: str, check_triggered: Optional[bool], email: Optional[str]):
+        self.runtime = runtime
+        self.email = email
+        self.version = version
+        self.major, self.minor = self.version.split(".")
+        self.check_triggered = check_triggered
+
+        self.redis_key = f"appdata:sast-scan:{self.version}"
+
+    async def set_last_event_redis(self, brew_event_id: str):
+        await redis.set_value(self.redis_key, brew_event_id)
+
+    async def get_last_event_redis(self):
+        return await redis.get_value(self.redis_key)
+
+    async def run(self):
+        cmd = [
+            "doozer",
+            "--group",
+            f"openshift-{self.version}",
+            "images:scan-osh",
+        ]
+
+        last_brew_event = await self.get_last_event_redis()
+
+        # Remove trailing /n
+        last_brew_event = last_brew_event.strip()
```

Comment:
nit: this would be better in the get method

Reply:
https://github.com/openshift-eng/art-tools/pull/51/commits/7c715b307aa02efa5d521ca0e931c9a2ae18766e